packages/server/lib/controllers/v1/stripe/postWebhooks.ts

```diff
@@ -42,37 +43,144 @@ export const postStripeWebhooks = asyncWrapper<PostStripeWebhooks>(async (req, r
         return;
     }
 
-    logger.info('[stripe-hook]', event.type);
+    logger.info('Received', event.type);
+    const handled = await handleWebhook(event);
+    if (handled.isErr()) {
+        report(handled.error, { body: req.body });
+        res.status(500).send({ error: { code: 'server_error', message: handled.error.message } });
+        return;
+    }
+
+    res.status(200).send({ success: true });
+});
 
+async function handleWebhook(event: Stripe.Event): Promise<Result<void>> {
```

Comment:
Same comments as the other webhook (idempotency and offload processing to queue) - more questions than expected changes for this PR though.

Reply:
answered above